internal/core/build_info.go

```diff
@@ -57,7 +57,9 @@ func (b *BuildInfo) GetUserAgent() string {
 }
 
 func (b *BuildInfo) checkVersion(ctx context.Context) {
-	if !b.IsRelease() || ExtractEnv(ctx, scwDisableCheckVersionEnv) == "true" {
+	cmd := extractMeta(ctx).command
```

Comment:
Could you add some documentation in the configuration file (cli.yaml) as well?

Reply:
Some documentation ? It is not configurable in `cli.yaml`.